src-python/env_set.md

```diff
@@ -0,0 +1,27 @@
+# Env Set up for Test
+
+This script is using Python3 (NOT 2 :-P)
+
+## 1. Check your Python Version and Binary Location
+```
+$ which python3
```

Comment:
needs to be python3.6 and higher
here is a link for install for ubuntu 16.04: https://askubuntu.com/questions/865554/how-do-i-install-python-3-6-using-apt-get